packages/payment-proxy-client/src/App.tsx

```diff
@@ -59,11 +59,9 @@ function Copyright(props: any) {
 }
 
 function computePercentagePaid(info: PaymentChannelInfo): number {
-  return (
-    Number(
-      info.Balance.PaidSoFar /
-        (info.Balance.RemainingFunds + info.Balance.PaidSoFar)
-    ) * 100
+  return Number(
+    (100n * info.Balance.PaidSoFar) /
```

Comment:
I don't think this is correct, since we're only multiplying the numerator by 100?